services/export/shared/init.js

```diff
@@ -0,0 +1,33 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2023 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const logger = require('lisk-service-framework').Logger();
+
+const { getCurrentChainID } = require('./helpers');
+
+const init = async () => {
+	try {
+		await getCurrentChainID();
```

Comment:
Do we need to cache it at init? This can be a hot reload when required. The entire `init` method can be removed.